src/Features/Core/Portable/EditAndContinue/EditAndContinueWorkspaceService.cs

```diff
@@ -41,6 +42,25 @@ public EditAndContinueWorkspaceService()
             _compilationOutputsProvider = GetCompilationOutputs;
         }
 
+        private static string? GetLogDirectory()
+        {
+            try
+            {
+                var path = Environment.GetEnvironmentVariable("Microsoft_CodeAnalysis_EditAndContinue_LogDir");
+                if (string.IsNullOrWhiteSpace(path))
+                {
+                    return null;
+                }
+
+                Directory.CreateDirectory(path);
```

Comment:
Does this throw if the path already exists?

Reply:
It does not.